### PR TITLE
[CFM-448] Moves block from the preprocess to a display suite field.

### PR DESCRIPTION
**Pull request by DavidHernandez**
Issue: https://issuetracker.amplexor.com/jira/browse/CFM-448

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/ds/c4m_ds/includes/c4m_ds.theme.user.inc [hunk 1]
```diff
@@ -106,3 +108,21 @@ function theme_c4m_ds_user_footer_stats(array $field) {
...
+ *   The field info.
+ *
+ * @return string
+ *   The rendered field.
+ */
+function theme_c4m_ds_user_my_groups(array $field) {
+  $block = block_load('c4m_user_profile', 'user_my_profile_groups');
+  $renderable_blocks = _block_render_blocks(array($block));
+  $renderable_array = _block_get_renderable_array($renderable_blocks);
+  return render($renderable_array);
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/ds/c4m_ds/includes/c4m_ds.theme.user.inc]: @DavidHernandez 
add a line spacing above this one.
(you can add `[ci skip]` to your commit message in order to skip travis)

File: project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my_groups.inc [hunk 2]
```diff
@@ -99,11 +99,25 @@ function c4m_features_homepage_get_group_type_title($type) {
 function c4m_features_homepage_get_user_groups($account = NULL) {
   $groups = c4m_user_og_get_groups($account);
   usort($groups, 'c4m_sort_object_by_title');
+  $groups = array_filter($groups, 'c4m_features_homepage_node_access_view_filter');
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my_groups.inc]: @DavidHernandez 
maybe this is better to do before the `usort()`
;)

**Comment by ordavidil** [comment]
@DavidHernandez 
any news about the behat test that fails ?

**Comment by DavidHernandez** [comment]
After re-rolling develop into my branch I'm able to reproduce the test failing locally. I'm trying to get it fixed.

**Pull request merged**